storybook/src/pages/amsterdam/common/AppHeader.tsx

```diff
@@ -1,13 +1,25 @@
-import { Grid, Header, PageMenu } from '@amsterdam/design-system-react'
-import { Alignment as PageMenuStory } from '../../../components/PageMenu/PageMenu.stories'
+import { Header, LinkList } from '@amsterdam/design-system-react'
 
 export const AppHeader = () => (
-  <Grid>
-    <Grid.Cell span="all">
-      <Header
-        links={<PageMenu alignEnd>{PageMenuStory.args?.children}</PageMenu>}
-        menu={<button className="ams-header__menu-button">Menu</button>}
-      />
-    </Grid.Cell>
-  </Grid>
+  <Header
+    noMenuButtonOnWideScreen
+    menuItems={[
+      <Header.MenuLink key="1" href="#" lang="en">
+        English
+      </Header.MenuLink>,
+      <Header.MenuLink key="2" href="#">
+        Mijn Amsterdam
+      </Header.MenuLink>,
+      <Header.MenuLink key="3" href="#" fixed>
+        Zoeken
+      </Header.MenuLink>,
+    ]}
+  >
+    <LinkList className="ams-mb--lg">
```

Comment:
Duplicate content might be an issue for public facing websites, especially when it's the main navigation component.

Reply:
Either the links in the header menu or those in the mega menu have `display: none`, which removes them from the accessibility tree.